<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_OgeWiFontConfig</class>
 <widget class="QWidget" name="C_OgeWiFontConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="stw::opensyde_gui_elements::C_OgeCbxFontProperties" name="pc_FontComboBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="stw::opensyde_gui_elements::C_OgeCbxText" name="pc_ComboBoxSize">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="stw::opensyde_gui_elements::C_OgePubToggleBold" name="pc_PushButtonBold">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="stw::opensyde_gui_elements::C_OgePubToggle" name="pc_PushButtonItalic">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="stw::opensyde_gui_elements::C_OgePubToggle" name="pc_PushButtonUnderline">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeCbxText</class>
   <extends>QComboBox</extends>
   <header>C_OgeCbxText.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeCbxFontProperties</class>
   <extends>QFontComboBox</extends>
   <header>C_OgeCbxFontProperties.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgePubToggleBold</class>
   <extends>QPushButton</extends>
   <header>C_OgePubToggleBold.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgePubToggle</class>
   <extends>QPushButton</extends>
   <header>C_OgePubToggle.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
